src/applications/personalization/profile/tests/e2e/direct-deposit/update-flow.cypress.spec.js

```diff
@@ -19,11 +19,17 @@ const dd4eduEnabled = {
   },
 };
 
-function fillInBankInfoForm() {
+function fillInBankInfoForm(id) {
```

Comment:
I added a test where _both_ edit bank info forms are visible at the same time so I needed a way to target _which_ bank info form is being filled in.